pkg/skaffold/instrumentation/meter.go

```diff
@@ -104,21 +104,23 @@ func init() {
 	}()
 }
 
-func InitMeter(runCtx *runcontext.RunContext, config *latest.SkaffoldConfig) {
+func InitMeter(runCtx *runcontext.RunContext, configs []*latest.SkaffoldConfig) {
 	meter.Command = runCtx.Opts.Command
-	meter.PlatformType = yamltags.GetYamlTag(config.Build.BuildType)
-	for _, artifact := range config.Pipeline.Build.Artifacts {
-		if _, ok := meter.Builders[yamltags.GetYamlTag(artifact.ArtifactType)]; ok {
-			meter.Builders[yamltags.GetYamlTag(artifact.ArtifactType)]++
-		} else {
-			meter.Builders[yamltags.GetYamlTag(artifact.ArtifactType)] = 1
-		}
-		if artifact.Sync != nil {
-			meter.SyncType[yamltags.GetYamlTag(artifact.Sync)] = true
+	meter.PlatformType = yamltags.GetYamlTag(configs[0].Build.BuildType)
+	for _, config := range configs {
+		for _, artifact := range config.Pipeline.Build.Artifacts {
+			if _, ok := meter.Builders[yamltags.GetYamlTag(artifact.ArtifactType)]; ok {
+				meter.Builders[yamltags.GetYamlTag(artifact.ArtifactType)]++
+			} else {
+				meter.Builders[yamltags.GetYamlTag(artifact.ArtifactType)] = 1
+			}
+			if artifact.Sync != nil {
+				meter.SyncType[yamltags.GetYamlTag(artifact.Sync)] = true
+			}
+			meter.Deployers = append(meter.Deployers, yamltags.GetYamlTags(config.Deploy.DeployType)...)
+			meter.BuildArtifacts += len(config.Pipeline.Build.Artifacts)
 		}
 	}
```

Comment:
```suggestion
		}
		meter.Deployers = append(meter.Deployers, yamltags.GetYamlTags(config.Deploy.DeployType)...)
		meter.BuildArtifacts += len(config.Pipeline.Build.Artifacts)
	}
```

Reply:
good catch! But this code block seems to have been removed from latest master